nextclade/resources/denv1/genome_annotation.gff3

```diff
@@ -0,0 +1,14 @@
+##gff-version 3
+##sequence-region NC_001477.1 1 10735
+NC_001477.1	feature	gene	95	394	.	+	.	codon_start=1;gene=C;gene_name=C;
+NC_001477.1	feature	gene	437	709	.	+	.	codon_start=1;gene=pr;gene_name=pr;
+NC_001477.1	feature	gene	710	934	.	+	.	codon_start=1;gene=M;gene_name=M;
```

Comment:
@j23414 Seems like [the errors you are seeing](https://github.com/nextstrain/dengue/pull/25#issuecomment-2133877167) stem from differences in the genome_annotation.gff3 and the reference GenBank file's annotations. For example, updating denv1's C and M gene annotations to match the annotations in in [reference_denv1_genome.gb](https://github.com/nextstrain/dengue/blob/bc07dac5d162a65e536bc718d44d881c78732b3a/phylogenetic/config/reference_denv1_genome.gb#L49-L57) fixes the denv1 test for me. 

```suggestion
NC_001477.1	feature	gene	95	436	.	+	.	codon_start=1;gene=C;gene_name=C;
NC_001477.1	feature	gene	437	709	.	+	.	codon_start=1;gene=pr;gene_name=pr;
NC_001477.1	feature	gene	437	934	.	+	.	codon_start=1;gene=M;gene_name=M;
```

Reply:
Thank you, this fixes denv1 test on my end! Interesting that `M` and `pr` are overlapping genomic regions, I must have manually copied the coordinates out wrong.

``` text
     CDS             437..934
                     /gene="M"
                     /product="membrane glycoprotein precursor M"
                     /protein_id="NP_733807.2"
     CDS             437..709
                     /gene="pr"
                     /note="peptide pr"
                     /product="protein pr"
                     /protein_id="YP_009164956.1"
```

Reply:
Thanks @joverlee521! 🙌 

That was the problem with the other datasets (e.g. `all`, `denv2`, `denv3`, `denv4`). Submitted fixups, I'm going to copy the datasets over to https://github.com/nextstrain/nextclade_data/pull/203